pkg/traefik/traefik.go

```diff
@@ -28,8 +29,13 @@ const (
 	image              = "traefik:v3.0"
 	appLabel           = "app"
 	appLabelValue      = "traefik"
-	replicas           = 2
+	replicas           = 1
 	waitRetry          = 5 * time.Second
+
+	defaultCPURequest    = "1000m"
+	defaultMemoryRequest = "512Mi"
+	maxCPULimit          = "2000m"
+	maxMemoryLimit       = "1024Mi"
```

Comment:
Do we need that much of resources?

Reply:
Not sure how much we need. How do you think we should configure it? or even make it configurable by the user?

Reply:
@jrmanes, what's your opinion here?